ui/src/app/components/runtime-configuration-panel/styles.tsx

```diff
@@ -142,4 +142,11 @@ export const styles = reactStyles({
   diskLabel: {
     fontWeight: 500,
   },
+  disabledCloudProfile: {
```

Comment:
I just noticed that cromwell is using the runtime-configuration-panel styles. I wonder if we want to either move this file to somewhere common or split out the app styles into its own file. I do not believe that this has to be done now.

Reply:
I agree thats separate PR, but i like the idea of moving this file to a common location